docs/quickstart/install-and-use-a-package-using-the-dotnet-cli.md

```diff
@@ -110,3 +110,9 @@ To explore more that NuGet has to offer, select the links below.
 - [Overview and workflow of package consumption](../consume-packages/overview-and-workflow.md)
 - [Finding and choosing packages](../consume-packages/finding-and-choosing-packages.md)
 - [Package references in project files](../consume-packages/package-references-in-project-files.md)
+
+## Related video
+
+> [!Video https://channel9.msdn.com/Series/NuGet-101/Install-and-Use-a-NuGet-Package-with-the-NET-CLI-3-of-5]
```

Comment:
You're missing `/player` at the end of the video URL. Additionally, next time please create 1 PR with all the changes. It reduces review workload.